addons/website_sale/views/templates.xml

```diff
@@ -234,7 +235,8 @@
                                                     <t t-set="product" t-value="td_product['product']" />
                                                     <td t-att-colspan="td_product['x'] != 1 and td_product['x']"
                                                         t-att-rowspan="td_product['y'] != 1 and td_product['y']"
-                                                        t-attf-class="oe_product #{td_product['class']}">
+                                                        class="oe_product"
```

Comment:
After discussion with JKE, as this has high chances of user customization + we customize it ourself and probably again in the future refactoring of shop, let's leave t-attf-class even if nothing is computed. Maybe just add a comment alongside.